ksql-common/src/main/java/io/confluent/ksql/schema/ksql/types/SqlStruct.java

```diff
@@ -34,6 +34,10 @@
 @Immutable
 public final class SqlStruct extends SqlType {
 
+  private static final String PREFIX = "STRUCT<";
+  private static final String POSTFIX = ">";
+  private static final String EMPTY_STRUCT = PREFIX + " " + POSTFIX;
```

Comment:
The space is necessary between `<` and `>` to stop the lexer seeing `<>` and interpreting as `NEQ`, i.e. not equal.